### PR TITLE
Cw98 manage attendees show users

### PR DESCRIPTION
**Pull request by LaurenSlu**
This started out as being manage employees but a few things snuck in.
this ticket is mainly comprised with:
1) displaying the users under each fitness class header
2) incredibly incorrect and shitty styling on said users (having some trouble)
3) non functional attend/not attend buttons ***
4) showing today's date when no to date is given
5) clearing out the attendees list when clear is pushed

the db states this flag is 'attended' or not but the buttons say attending...that's confusing...like...currently attending? i dunno, we might want to talk about the wording a bit

### CODE REVIEW

**Comment by Yexur** [comment]
still has a merge conflict on dateUtils.ts

File: src/api/export.api.ts [hunk 1]
```diff
@@ -4,6 +4,6 @@ export * from './instructor_api';
 export * from './location_api';
 export * from './registration_api';
 export * from './user_api';
-export * from './fitnessClass_api';
+export * from './fitnessClass_api'
```
Comment (Yexur) [inline src/api/export.api.ts]: keep the semicolon

File: src/api/manageAttendees_api.ts [hunk 2]
```diff
@@ -45,9 +46,33 @@ export class ManageAttendeesAPI{
...
+                data.endTime = DateUtils.convertStringToTime(data.endTime, false);
                 const id = fClass.payload.doc.id;
                 return {id, ...data};
             });
         });
     }
+
+    getAttendeesForClass(fitnessClassId: string): Observable<ManageAttendeesModel[]> {
+        let attendeeCollection: AngularFirestoreCollection<ManageAttendeesData>;
+        let docPath = FITNESS_CLASSES + "/" + fitnessClassId + "/" + REGISTRATIONS; //check with mike if this is even right
```
Comment (Yexur) [inline src/api/manageAttendees_api.ts]: its right.  please remove the comment

File: src/models/listOfClassAttendeesModel.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {FitnessClassModel } from './export.models';
```
Comment (Yexur) [inline src/models/listOfClassAttendeesModel.ts]: This import statement is not needed

**Comment by LaurenSlu** [comment]
@Yexur  i dont know how to reply to your above comment...i feel like an old person using technology

i solved the merge conflict :)

File: src/pages/manage-attendees/manage-attendees.scss [hunk 4]
```diff
@@ -35,4 +35,20 @@ page-manage-attendees {
...
+
+    .attendee-row:nth-child(even) {
+        background: map-get($colors, grey);;
+    }
+
+    .attendee-row:nth-child(odd) {
+        background: map-get($colors, white);;
+    }
+    
+    .attendee-name {
```
Comment (Yexur) [inline src/pages/manage-attendees/manage-attendees.scss]: are these place holders?

Comment (LaurenSlu) [reply src/pages/manage-attendees/manage-attendees.scss]: @Yexur yeah i think Ill end up using them. I see Josh had some in his branch that were empty and you had approved it so I assume you were cool with it

Comment (kjrcda) [reply src/pages/manage-attendees/manage-attendees.scss]: oops i must have missed those lol

File: src/shared/dateUtils.ts [hunk 5]
```diff
@@ -35,7 +35,6 @@ export module DateUtils {
         return date.format('YYYY-MM-DD');
     }
 
-    export function convertStringToTime(timeObj: string): string {
```
Comment (kjrcda) [inline src/shared/dateUtils.ts]: removed the function declaration but not the function?

Comment (LaurenSlu) [reply src/shared/dateUtils.ts]: this was a merge conflict issue. my bad.

Comment (LaurenSlu) [reply src/shared/dateUtils.ts]: wait Im confused what's even happening here....that function should still be there...Im using it....

File: src/pages/manage-attendees/manage-attendees.scss (hunk 4 above)
Comment (Yexur) [reply src/pages/manage-attendees/manage-attendees.scss]: I missed those as well... I would have commented on it.... my bad

Comment (LaurenSlu) [reply src/pages/manage-attendees/manage-attendees.scss]: removed.

**Pull request merged**